doc/examples/particles/README.md

```diff
@@ -0,0 +1,4 @@
+# render_flip
+
+A Flame game showcasing how to use render flipping on a `PositionComponent`
+to control the rendered direction.
```

Comment:
probably forgot to update this file ;)

Reply:
Yup, as well as tests and submission for website docs, will work on that after we settled on overall approach :)